packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -499,6 +523,19 @@ Future<void> buildGradleApp({
     '$successMark Built ${globals.fs.path.relative(apkFile.path)}$appSize.',
     color: TerminalColor.green,
   );
+
+  if (buildInfo.analyzeAotSize) {
+    final Map<String, dynamic> apkAnalysisJson = SizeAnalyzer().analyzeApkSize(
+      apk: apkFile,
+      aotSizeJson: aotSizeJsonFile,
+    );
+    final File apkAnalysisFile = apkDirectory.childFile('apk-analysis.json');
+    apkAnalysisFile.writeAsStringSync(json.encode(apkAnalysisJson));
+    // TODO(peterdjlee); Output a clickable link that launches devtools and automatically loads the data from analyze.json.
```

Comment:
outputting a clickable link would be hard as flutter_tools should be done running. it could make more sense to instead have a flag (perhaps the default) for this tool that launches devtools automatically opening this json file.